charts/backstage/values.schema.tmpl.json

```diff
@@ -143,6 +143,14 @@
                         "backstage.10.0.0.1.nip.io"
                     ]
                 },
+                "path": {
+                    "title": "Path to be used to expose the full route to access the backstage application.",
+                    "type": "string",
+                    "default": "/",
+                    "examples": [
```

Comment:
Wouldn't the example be `/backstage` instead of the host and path?

Reply:
Hmm, you are right, let me change it.

Reply:
Ok done. Thanks.